provider/openstack/local_test.go

```diff
@@ -864,7 +864,10 @@ func (s *localServerSuite) TestStartInstanceDeletesSecurityGroupsOnInstanceCreat
 	c.Check(inst, gc.IsNil)
 	c.Assert(err, gc.NotNil)
 
-	assertSecurityGroups(c, env, []string{"default"})
+	assertSecurityGroups(c, env, []string{
```

Comment:
**suggestion:**  Same as below, use `s.Tests.ControllerUUID` to generate the security group name related to controller

Reply:
Done